setup.py

```diff
@@ -46,7 +46,7 @@ def package_files(directory):
 
 
 version_file_path = path.join("src", "cocotb", "_version.py")
-__version__ = "2.0.0"
+__version__ = "2.1.dev0"
```

Comment:
We've used the 3 digit form up till now.
```suggestion
__version__ = "2.1.0.dev0"
```

Reply:
Semver is always three digits.